gtsam/geometry/Cal3Bundler.cpp

```diff
@@ -118,6 +121,25 @@ Point2 Cal3Bundler::calibrate(const Point2& pi, const double tol) const {
     throw std::runtime_error(
         "Cal3Bundler::calibrate fails to converge. need a better initialization");
 
+  x = invKPi.x(), y = invKPi.y();
+  const double xx = x * x, yy = y * y;
+  const double rr = xx + yy;
+  const double g = (1 + k1_ * rr + k2_ * rr * rr);
+  const double inv_f = 1 / f_, inv_g = 1 / g;
+
+  // Approximate the jacobians via a single iteration of g.
```

Comment:
Obsolete comment if you buy my argument above, but you are still using the initial estimate for jacobians?